### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by joyaljaimon**
Hi

My name is Joyal Jaimon.I am an English Professor and I have read through your documentation.Because of the sheer amount of pull requests I think the documentation is reaching perfection.I have found very little changes which could enhance the experience for the end user.Kindly go through it hope it helps.

Thanks And Regards

Keep Coding
Joyal Jaimon

### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Contributing to Kong :monkey_face:
 
 Hello, and welcome! Whether you are looking for help, trying to report a bug,
-thinking about getting involved in the project, or about to submit a patch, this
+thinking about getting involved in our projects, or about to submit a patch, this
```
Comment (kikito) [inline CONTRIBUTING.md:4]: Please change this back, this document only concerns itself with a single project - the Kong Gateway.

```suggestion
thinking about getting involved in the project, or about to submit a patch, this
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 document is for you! It intends to be both an entry point for newcomers to
-the community (with various technical backgrounds), and a guide/reference for
+the community (with various technical backgrounds), and a guide/reference too for
```
Comment (kikito) [inline CONTRIBUTING.md:6]: I don't think we should use "too" here.
```suggestion
the community (with various technical backgrounds), and a guide/reference for
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -32,23 +32,23 @@ Consult the Table of Contents below, and jump to the desired section.
 
 ### Enterprise Edition
 
-If you are a Kong Enterprise customer, contact the Enterprise Support channels
+If you are a Kong Enterprise customer, you can contact the Enterprise Support channels
```
Comment (kikito) [inline CONTRIBUTING.md:35]:
```suggestion
If you are a Kong Enterprise customer, you may contact the Enterprise Support channels
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -32,23 +32,23 @@ Consult the Table of Contents below, and jump to the desired section.
...
 by opening an Enterprise support ticket on
 [https://support.konghq.com](https://support.konghq.com/).
 
-If you are experiencing a P1 issue, please call the [24/7 Enterprise Support
+If you are experiencing a P1 issue, kindly call us at the [24/7 Enterprise Support
```
Comment (kikito) [inline CONTRIBUTING.md:39]: Please change this back. The Enterprise Support team isn't necessarily the same as "Us", the authors of this document. They are related but not exactly the same thing.

```suggestion
If you are experiencing a P1 issue, please call at the [24/7 Enterprise Support
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -32,23 +32,23 @@ Consult the Table of Contents below, and jump to the desired section.
...
 phone line](https://support.konghq.com/hc/en-us/articles/115004921808-Telephone-Support)
 for immediate assistance, as published in the Customer Success Reference Guide.
 
-If you are interested in becoming a Kong Enterprise customer, please visit
+If you are interested in becoming a Kong Enterprise customer, feel free to visit
```
Comment (kikito) [inline CONTRIBUTING.md:43]: Kindly change this back, please. That particular phrase was carefully chosen.

```suggestion
If you are interested in becoming a Kong Enterprise customer, please to visit
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -130,11 +130,11 @@ We **do not** accept new plugins into the core repository. The plugins that are
 currently part of this repository are there because of historical reasons, but
 will be pushed into separate repositories in the foreseeable future.
 
-If you wish to write a new plugin for your own needs, you should start by
+If you are interested in writing a new plugin for your own needs, you should begin by
 reading the [Plugin Development
 Guide](https://docs.konghq.com/latest/plugin-development).
 
-If you already wrote a plugin, and are thinking about making it available to
+If you have already wrote a plugin, and are thinking about making it available to
```
Comment (kikito) [inline CONTRIBUTING.md:137]: This phrase is broken, please change it back.
```suggestion
If you already wrote a plugin, and are thinking about making it available to
```

**Review by kikito — COMMENTED** [review]
Hello Joyal, thank you for your contribution. Indeed we have had a lot of contributions to our docs.

I have requests on your PR. Could you please make the amendments I require?

**Comment by joyaljaimon** [comment]
Hi

Thanks for responding.I have made the necessary changes as requested.Kindly merge the pull request.

Thank You for considering my contribution.

Joyal Jaimon

**Review by kikito — APPROVED** [review]

**Comment by kikito** [comment]
Merged, thanks!

**Pull request merged**